.github/workflows/ci.yml

```diff
@@ -272,6 +287,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/setup-zeebe
+        with:
```

Comment:
:x: This is running on github provided runners, using our own nexus does not make sense IMO. We can also see that the go client tests take much longer to complete now, apparently because pulling from nexus takes more time than pulling from maven central.